README.md

```diff
@@ -74,6 +75,23 @@ pip install -t <install location> .
 ``` 
 The -t option specifies an install location which can be useful if you are on a computing cluster and don't have write access to the default install location. If you specify a non-standard location you will need to add it to your `PYTHONPATH` as above so that python can find the module.
 
+
+### Sphinx Documentation
+
+The pyMACh3 module is documented using sphinx. Currently you need to build this documentation yourself if you want it (in future this will be automated). To do this, you will need to install Mach3 with its python extension as described above, then go to the [Doc/sphinx](Doc/sphinx) directory. Then you will need to install sphinx and the necessary extensions which can be done using 
+
+```
+pip install -r requirements.txt
+```
+
+then you can simply do 
+
+```
+make html
+```
+
+and the documentation will be built in the build/html directory which you can open with whatever browser you like.
+
```

Comment:
can you move this readme somehwere else? for example to Doc/spinx?

If you are planning to make this autgoerated then maye not add this to main readme?

I worry this will confuse people.